CHANGELOG.rst

```diff
@@ -2,6 +2,23 @@
 Release Notes
 =============
 
+6.12.2
+======
+
+Prelude
+-------
+
+Release on: 2019-07-03
+
+This release is only available on Windows.
+
+- Please refer to the `6.12.2 tag on integrations-core <https://github.com/DataDog/integrations-core/blob/master/AGENT_CHANGELOG.md#datadog-agent-version-6122>`_ for the list of changes on the Core Checks
+
+Bug Fixes
+---------
+
+- Avoid a bug on windows 2016 that caused memory leaks in checks using WMI.
```

Comment:
it's weird to mention a fix for a bug that was never shipped, so I'd remove this altogether